deb/tarsnap-archive-keyring/Makefile

```diff
@@ -4,30 +4,37 @@ all: tarsnap-archive-keyring.gpg tarsnap-code-signing-keyring.gpg tarsnap-experi
 # certain package-building systems, which might not have a valid user home
 # directory within its chroot or jail.
 TMP_GPGHOME=gpg-tmp
+TMP_GPGHOME_CREATED=gpg-tmp-created
 GPG=gpg --homedir=$(TMP_GPGHOME) --no-default-keyring --no-auto-check-trustdb
 
-$(TMP_GPGHOME):
+# We only want to create $(TMP_GPGHOME) once, but that directory's metadata
+# changes when files are created within it; that can lead to re-creating the
+# directory, and then re-creating all the keyrings.  To avoid this, we use
+# the $(TMP_GPGHOME_CREATED) file to store the creation time so that `make`
+# does what we want it to.
+$(TMP_GPGHOME_CREATED):
 	mkdir -p $(TMP_GPGHOME)
 	chmod 700 $(TMP_GPGHOME)
+	touch $(TMP_GPGHOME_CREATED)
```

Comment:
I feel like this is a common idiom, and should have a well-known name.  A "timestamp file"?  "flag file"?  "semaphore file"?

If there's a common name, I should revise the comment to use it.

Reply:
"Sentinel file".

FWIW, I would put the sentinel file inside the home directory -- it's more obviously correct since there's no way the file can exist without the directory which contains it also existing.